processor/compile/image.js

```diff
@@ -7,11 +7,6 @@ module.exports = function ImageCompiler() {
   visitors.image = function compile(node, ...args) {
     if (node.data?.hProperties?.className === 'emoji') return node.title;
 
-    // Use magic block instead of markdown when there are certain defined properties we can't store in markdown
-    const { align, className, width } = node.data?.hProperties || {};
-    const useMagicBlock = Boolean(width) || className?.length || Boolean(align);
-    if (useMagicBlock) return visitors.figure.call(this, node, ...args);
```

Comment:
I _think_ we'll need to replace this something that writes out a jsx tag, right? Something like:

```
return `<Image width="${width}" />`
```

Reply:
it didn't seem to be an issue? we're not having to store the extra properties in a magic block anymore, so the component just spits out the correct JSX appropriately (unless I've gotten all this wrong, which is also entirely possible 😂 )

Reply:
The compilers take an `ast` and spit out `md/mdx`. This mostly happens when the editor saves, so I'm pretty sure saving images with options won't save the extra options until we replace the magic block stuff.

Reply:
how do i test this? are we able to use the RDMX flag in the main app yet?

Reply:
This [file](https://github.com/readmeio/markdown/blob/next/__tests__/flavored-compilers/images.test.js#L4) really should have had a magic block test.

And yea, the flag ~works. A lot of things can crash the page.